test/Microsoft.AspNetCore.SignalR.Redis.Tests/EchoHub.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.SignalR.Redis.Tests
+{
+    public class EchoHub : Hub
+    {
+        public override Task OnConnectedAsync()
+        {
+            return Groups.AddAsync(Context.ConnectionId, "Test");
+        }
+
+        public override Task OnDisconnectedAsync(Exception exception)
+        {
+            return Groups.RemoveAsync(Context.ConnectionId, "Test");
```

Comment:
Do you **need** to do this or you **want** to do this? A connection should be removed from a group automatically when it disconnects

Reply:
Good point, I should check that. Also I'm probably going to add a hub method that assigns yourself to a group instead of doing it in `OnConnectedAsync`